ltp-production.yaml

```diff
@@ -236,14 +223,7 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=3318
     active: true
     intermittent: false
-  - environments:
-    - hi6220-hikey
-    - juno-r2
-    - qemu_x86_64
-    - qemu_arm64
-    - dragonboard-410c
-    - qemu_i386
```

Comment:
Was it intentional to drop i386 from this known issue? I noticed when reading the build output @ https://travis-ci.com/Linaro/qa-reports-known-issues/builds/84374661

Reply:
Good catch !
I have intentionally dropped i386 because it is getting pass on i386 confirmed from staging results.